libnavigation-core/src/test/java/com/mapbox/navigation/core/reroute/MapboxRerouteControllerTest.kt

```diff
@@ -121,7 +130,7 @@ class MapboxRerouteControllerTest {
         verify(exactly = 2) {
             tripSession.locationMatcherResult
         }
-        verify(exactly = 1) {
+        verify(exactly = 2) {
```

Comment:
what is the point of counting `getRouteProgress` calls? Should the test become red indicating that something doesn't work when we have 3 calls?

Reply:
we check that method is invoked, mockk provides a 0-cost feature with `exactly`, so it gives a chance to catch regression with no additional effort